core/services/relay/evm/functions/threshold_config_poller.go

```diff
@@ -0,0 +1,167 @@
+package functions
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/libocr/gethwrappers2/ocr2aggregator"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2/types"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+)
+
+type ThresholdConfigPoller interface {
```

Comment:
Why not FuncionsConfigPoller that internally has a flag target_config={FUNCTIONS, THRESHOLD, S4} ?